tests/models/test_openai.py

```diff
@@ -2929,3 +3028,64 @@ def test_deprecated_openai_model(openai_api_key: str):
 
         provider = OpenAIProvider(api_key=openai_api_key)
         OpenAIModel('gpt-4o', provider=provider)  # type: ignore[reportDeprecated]
+
+
+@pytest.mark.vcr()
+async def test_openai_video_url_raises_not_implemented(openai_api_key: str, allow_model_requests: None) -> None:
+    model = OpenAIChatModel('gpt-4o', provider=OpenAIProvider(api_key=openai_api_key))
+    with pytest.raises(NotImplementedError):
+        await model.request(
+            [ModelRequest(parts=[UserPromptPart(content=[VideoUrl(url='https://example.com/file.mp4')])])],
+            {},
+            ModelRequestParameters(),
+        )
+
+
+async def test_openai_map_single_item_unknown_returns_empty_branch(
```

Comment:
I'm not sure why we need this test and the next one. What lines would be uncovered without them?